### PR TITLE
Add install dependecy on regions

### LINKED ISSUE #139: Add dependency on astropy regions
Currently when mosviz is installed it does not have a dependency on astropy regions, which is required.

### PR DESCRIPTION
**Pull request by drdavella**
This resolves #139.

### CODE REVIEW

**Comment by drdavella** [comment]
There's currently a note in the travis config that says we require an unreleased version of `regions`, so I'm not sure how to handle this. Will a new version be released soon? Or should we use the github dependency in our `setup.cfg`?

**Comment by larrybradley** [comment]
`regions 0.3` was release last week:  https://groups.google.com/d/msg/astropy-dev/dv7b8L-FnoE/qQ3O7LRMCAAJ

**Comment by drdavella** [comment]
@larrybradley I'm not sure if that's sufficiently recent since the release happened on 9/8 but the comment is from 9/12.

@robelgeda since you left the comment, was the functionality we require released in 0.3, or is it still unreleased?

**Comment by larrybradley** [comment]
@drdavella  Hmm, there's only been one very minor PR merged (https://github.com/astropy/regions/pull/222) since the release. 🤔

**Comment by drdavella** [comment]
Okay, I'll try adding `>=0.3` and removing it from the pip requirements in the travis matrix to see what happens.

**Comment by larrybradley** [comment]
The travis failure is because you haven't installed the regions package for the doc-build test.

**Comment by drdavella** [comment]
@larrybradley I'm thinking that I should just add regions back to `PIP_DEPENDENCIES` then.

**Comment by larrybradley** [comment]
@drdavella Yes, that should work.

Is `regions` an optional dependency?  I don't see it in the setup:  https://github.com/spacetelescope/mosviz/blob/master/setup.cfg#L50

If it's an optional dependency, I would recommend adding a travis test where you *don't* install it (or any other optional dependencies) to check for import errors.

File: appveyor.yml [hunk 1]
```diff
@@ -13,9 +13,7 @@ environment:
 
       NUMPY_VERSION: "stable"
       CONDA_DEPENDENCIES: "pyqt numpy astropy glue-core>=0.13 specviz=0.5"
-
-      # N.B regions package: need the work in master as of 9-11-2018 which has not yet been released
-      PIP_DEPENDENCIES: "pytest-qt git+https://github.com/astropy/regions.git"
+      PIP_DEPENDENCIES: "pytest-qt"
```
Comment (astrofrog) [inline appveyor.yml]: Why not include ``regions>=0.3`` here?

Comment (drdavella) [reply appveyor.yml]: Gah, an oversight. Although, to be fair, my original intent was to test the installation of dependencies during `setup.py develop/install`, which very few of our packages actually do...

**Comment by robelgeda** [comment]
@drdavella This looks good to me V0.3 is what we need.

**Pull request merged**